### ISSUE TITLE
API Documentation Blank

### ISSUE THREAD

**Issue by tiicon** [benjipott/video.js-chromecast#15]
Hi there,

Looking to utilize this in a project of mine, but the API documentation is completely blank. Any chance of this being updated soon with documentation?

Regards,

-TC

**Comment by Krato** [comment]
Yes please. It seems a good plugin but no idea how to implement it.

Thanks.

**Comment by Krato** [comment]
Why you close this issue? The api documentation still blank.

**Comment by benjipott** [comment]
All the documentation has updated yesterday and is on readme file
